hyperliquid/info.py

```diff
@@ -187,6 +187,37 @@ def user_fills(self, address: str) -> Any:
         """
         return self.post("/info", {"type": "userFills", "user": address})
 
+    def user_fills_by_time(self, address: str, startTime: int, endTime: Optional[int] = None) -> Any:
```

Comment:
nit: please use snakecase for variables in python, e.g. `startTime -> start_time` and be sure to change all references to this variables to snakecase in the docs below as well